### ISSUE TITLE
miktex-mf (or maketfm) doesn't find .mf files in the current directory

### ISSUE THREAD

**Issue by shreevatsa** [MiKTeX/miktex#672]
- [x] I have read the [MiKTeX Contributing Guidelines](https://github.com/MiKTeX/miktex/blob/master/CONTRIBUTING.md)

Steps to reproduce:

1. Create a file called, say, `cee.mf` (ideally a valid METAFONT file, but to illustrate this issue the contents don't matter as the file won't be found anyway).

2. In the same directory, create a `.tex` file containing:

   ```
   \documentclass{article}  
   \begin{document}
   \font\ceefont=cee
   Hey look I can draw some Cs: {\ceefont ccc}.
   \end{document}
   ```

3. Now run `pdflatex` on the `.tex` file. The output will be something like:

   ```
   Making cee.tfm from ./cee.mf...
   Running miktex-mf...
   This is METAFONT, Version 2.7182818 (MiKTeX 20.11)
   
   ! I can't find file `cee'.
   <*> \mode:=ljfour; nonstopmode; input cee
   
   Please type another input file name
   ! Emergency stop.
   <*> \mode:=ljfour; nonstopmode; input cee
   
   Transcript written on mfput.log.
   METAFONT failed for some reason
   
   Sorry, but maketfm did not succeed.
   
   The log file hopefully contains the information to get MiKTeX going again:
   ```

   etc.

In other words, `miktex-mf`, invoked by `maketfm`(I think?) via `pdflatex`, does not find the `.mf` file that is in the same directory. My guess is that the command runs in some temporary directory, maybe.

For what it's worth, this issue was previously reported in 2011 [on Sourceforge](https://sourceforge.net/p/miktex/bugs/2079/), based on a question [on StackExchange](https://tex.stackexchange.com/q/25959). I encountered this on macOS but I believe the issue must be global; let me know if further information about my MiKTeX installation is necessary.

**Comment by shreevatsa** [comment]
A probably related point is that the generated `tfm` and `pk` files go into (on macOS):

```
~/Library/Application Support/MiKTeX/texmfs/data/fonts/tfm/cg/egyptian/cee.tfm
~/Library/Application Support/MiKTeX/texmfs/data/fonts/pk/ljfour/cg/egyptian/dpi600/cee.pk
```

(Why `egyptian`?) With TeX Live, they go into the current directory (same as the `.mf` file).

**Comment by edocevoli** [comment]
`maketfm` installs fonts according to the `fontname` nomenclature. See here: https://www.ctan.org/pkg/fontname

**Comment by shreevatsa** [comment]
Ah thanks, I didn't know that. So, here, because I named the file `cee.mf`, the first character `c` indicated Compugraphics (`cg`), and the last two `ee` indicated style `egyptian`. Makes sense!

BTW, here is the minimal/self-contained `cee.mf` I was using, in case it helps:

```
mode_setup;

% (character code, width, height, depth)
beginchar("c", 8pt#, 10pt#, 0);
  % Can use `w`, `h` and `d` (=0 here).
  z1 = (1/7 w, 7/8 h);
  z2 = (6/7 w, 4/8 h);
  z3 = (3/7 w, 3/8 h);
  z4 = (3/7 w, 1/8 h);
  pickup pencircle scaled 0.15w;
  draw z1 .. z2 .. z3 -- z4;
  penlabels(1, 2, 3, 4);  
endchar;

end
```

**Comment by edocevoli** [comment]
This issue has been fixed.
